build.sh

```diff
@@ -55,11 +55,12 @@ sbt installM2
 sbt test
 sbt pkgServer
 sbt serv/test
-cp -av "serv/target/universal/"*.zip ../../client/ts
```

Comment:
@scholarsmate This script was failing for me locally. Can you add In this location:

```bash
unzip serv/target/universal/*.zip -d ../../client/ts
```

then remove the `unzip -fo` command below this. That fixed my issue locally

Reply:
What failed exactly?

Reply:
Okay, I think it's the -fo switches which I thought would just overwrite/refresh what was there already.  I'll nuke that and I think it'll work.

Reply:
Using the online editor, I removed the f from the -fo.  Give that a try.